ui-events-winit/src/lib.rs

```diff
@@ -17,6 +17,351 @@
 #![no_std]
 
 pub mod keyboard;
+pub mod pointer;
+
+extern crate alloc;
+use alloc::{vec, vec::Vec};
+
+extern crate std;
+use std::time::Instant;
+
+use ui_events::{
+    keyboard::KeyboardEvent,
+    pointer::{PointerEvent, PointerId, PointerInfo, PointerState, PointerType, PointerUpdate},
+};
+use winit::{
+    event::{ElementState, Force, Touch, TouchPhase, WindowEvent},
+    keyboard::ModifiersState,
+};
+
+/// Struct for stateful transformations of winit [`WindowEvent`].
```

Comment:
Don't need to say that this is a struct. Should say a bit more about how it is used.

Reply:
Also, add a mention of this to the crate level doc at the top of the file